db/cache/charts.go

```diff
@@ -1010,80 +1044,168 @@ func avgBlockTimes(ticks, blocks ChartUints) (ChartUints, ChartUints) {
 }
 
 func blockSizeChart(charts *ChartData, bin binLevel, axis axisType) ([]byte, error) {
+	seed := binAxisSeed(bin, axis)
 	switch bin {
 	case BlockBin:
-		return charts.encode(charts.Blocks.Time, charts.Blocks.BlockSize)
+		switch axis {
+		case HeightAxis:
+			return encode(lengtherMap{
+				sizeKey: charts.Blocks.BlockSize,
```

Comment:
Is `heightKey` not needed here because it's implicitly height for `BlockBin`?

Reply:
Right.  Just for block-binned data.